tensorcircuit/translation.py

```diff
@@ -18,6 +18,7 @@
     from qiskit.circuit.quantumcircuitdata import CircuitInstruction
     from qiskit.circuit.parametervector import ParameterVectorElement
     from qiskit.circuit import Parameter, ParameterExpression
+    import cirq
```

Comment:
catch cirq import in another ``try..except...`` block? since the except warning below is for qiskit

Reply:
Sure, already changed